global/sass-config/_mixins.sass

```diff
@@ -25,3 +25,7 @@
 =size-square($size)
   height: $size
   width: $size
+
+// Slide
+=slide($perc)
```

Comment:
Always use full names for variables.
The typing is not the part of programming that takes the majority of the time. It's reading and updating existing code. So the more we can do to improve clarity to the reader the better :)

Reply:
Will remember that!